tests/test_api.py

```diff
@@ -279,6 +279,30 @@ def test_record2marcxml_handles_repeated_subfields():
     assert expected == result
 
 
+def test_record2marcxml_strips_control_characters():
+    record = {
+        '$schema': 'http://localhost:5000/schemas/records/hep.json',
+        'abstracts': [
+            {
+                'source': 'submitter',
+                'value': u'A common feature shared by many quantum gravity models is modi\u001Ccations of two-point functions at energy scales around the Planck scale.',
```

Comment:
With @michamos we saw that putting directly the control character wasn't showing up in his browser, so I replaced it with its unicode code point.